public/variables.css

```
@@ -7,31 +7,31 @@
   --font-serif: "PT Serif", serif;
   --font-mono: "PT Mono", monospace;
 
-  --neutral-0: #ffffff;
-  --neutral-50: #fefefe;
-  --neutral-100: #efefef;
-  --neutral-200: #dcdcdc;
-  --neutral-300: #bdbdbd;
-  --neutral-400: #989898;
+  --neutral-0: light-dark(#ffffff, #000000);
+  --neutral-50: light-dark(#fefefe, #0a0a0a);
+  --neutral-100: light-dark(#efefef, #3d3d3d);
+  --neutral-200: light-dark(#dcdcdc, #464646);
+  --neutral-300: light-dark(#bdbdbd, #525252);
+  --neutral-400: light-dark(#989898, #656565);
   --neutral-500: #7c7c7c;
```

```suggestion
  --neutral-500: light-dark(#7c7c7c, #7c7c7c);
```
**Nitpick:** Just for consistencies sake.